benchmark/_http-benchmarkers.js

```diff
@@ -133,20 +136,19 @@ exports.run = function(options, callback) {
     benchmarker: exports.default_http_benchmarker
   }, options);
   if (!options.benchmarker) {
-    callback(new Error('Could not locate required http benchmarker. See ' +
-                       'https://github.com/nodejs/node/blob/master/doc/guides/writing-and-running-benchmarks.md##http-benchmark-requirements ' +
-                       'for further instructions.'));
+    callback(new Error(`Could not locate required http benchmarker. See ${
+                                  requirementsURL} for further instructions.`));
```

Comment:
I've tried to stress the subordinate state of the next part. I'll try to decrease indentation.